.github/workflows/build-previews.yml

```diff
@@ -18,6 +18,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Make create_redirects.sh executable
+        run: chmod +x ./.github/workflows/create-redirects.sh
```

Comment:
You shouldn't need to do this here, just mark the sh file as executable on command line locally and commit the change:
```
chmod +x ./.github/workflows/create-redirects.sh
git commit -a
```